packages/kit/src/views/Earn/hooks/usePortfolioAction.ts

```
@@ -0,0 +1,308 @@
+          break;
+        case 'listaCheck':
+          void handleListaCheckAction({
+            stakedSymbol: stakedSymbol || symbol,
+            rewardSymbol,
+          });
+          break;
+        default:
+          break;
+      }
+    },
+    [handleClaimAction, handleClaimWithKycAction, handleListaCheckAction],
```

ESLint

[error] React Hook useCallback has a missing dependency: 'symbol'. Either include it or remove the dependency array.

(react-hooks/exhaustive-deps)